src/blocktypes/proviso/publish.sh

```diff
@@ -0,0 +1,29 @@
+SKOPEO_INSPECT=${SKOPEO_INSPECT:-$(dirname "${BASH_SOURCE[0]}")/skopeo-inspect.sh}
+
+function proviso() {
+  local -n input=$1
+  local -n output=$2
+
+  local missing tag
+  local -a images filtered
+
+  missing=$(mktemp)
+  trap 'rm -f $missing' RETURN
+
+  mapfile -t images < <(jq -r '.meta.images[0]|select(.!=null)' <<<"${input[@]}")
+
+  echo "${images[@]}" |
+    command xargs -n 1 -P 0 "$SKOPEO_INSPECT" "$missing"
+
+  for in in "${input[@]}"; do
+    tag=$(jq -r '.meta.images[0]' <<<"$in")
```

Comment:
Now we have an `image`, here, too. :smile:

Reply:
oh whoops